src/main/scala/chisel3/util/Conditional.scala

```diff
@@ -10,6 +10,7 @@ import scala.language.experimental.macros
 import scala.reflect.macros.blackbox._
 
 import chisel3._
+import chisel3.internal.sourceinfo.{SourceInfo}
```

Comment:
nit: Brackets are unnecessary.

```suggestion
import chisel3.internal.sourceinfo.SourceInfo
```